### PR TITLE
[FEATURE] Add play_digits helper

### PR DESCRIPTION
**Pull request by chewi**
Hi Ben. Here is play_digits as promised. Specs added, all pass.

### CODE REVIEW

**Comment by benlangfeld** [comment]
I'm not sure we really need the type check here, or indeed in the method it was borrowed from. Can you remove them before I merge?

**Comment by chewi** [comment]
I take it you want the regexp check gone as well? Because 1234 =~ /^\d+$/ returns nil.

**Comment by benlangfeld** [comment]
Please :)

File: lib/adhearsion/asterisk/call_controller_methods.rb [hunk 1]
```diff
@@ -362,9 +362,15 @@ def play_time(*args)
       # @param [Numeric|String] Numeric argument, or a string contanining numbers.
       # @return [Boolean] Returns false if the argument could not be played.
```
Comment (benlangfeld) [inline lib/adhearsion/asterisk/call_controller_methods.rb]: This is now incorrect. Please remove.

File: lib/adhearsion/asterisk/call_controller_methods.rb [hunk 2]
```diff
@@ -362,9 +362,15 @@ def play_time(*args)
...
-        if argument.kind_of?(Numeric) || argument =~ /^\d+$/
-          execute "SayNumber", argument
-        end
+        execute "SayNumber", argument
+      end
+
+      #Executes SayDigits with the passed argument.
+      #
+      # @param [Numeric|String] Numeric argument, or a string contanining numbers.
+      # @return [Boolean] Returns false if the argument could not be played.
```
Comment (benlangfeld) [inline lib/adhearsion/asterisk/call_controller_methods.rb]: This is now incorrect. Please remove.

**Comment by benlangfeld** [comment]
Couple of doc fixes required and then I'll merge :)

**Pull request merged**